documentation/src/main/markdown/gettingstarted/gettinghelp.md

```diff
@@ -1,13 +1,13 @@
 # Command line help options
 
-[company_name] [solution_name] provides the following execution options to assist with configuration, execution and support.
+[detect_product_long] provides the following execution options to assist with configuration, execution and support.
 
 | Mode | Command line option | Alt. option | Description |
 | ---- | ------------------- | ----------- | ----------- |
 | Help | --help | -h | Provides basic help information (including how to get more detailed help). |
-| Interactive | --interactive | -i | Guides you through configuring [company_name] [solution_name]. |
+| Interactive | --interactive | -i | Guides you through configuring [detect_product_short]. |
 | Diagnostic | --diagnostic | -d | Creates a zip file containing diagnostic information for support.  <br /> See [Diagnostic Mode](../troubleshooting/diagnosticmode.md)|
 | hyaml | --helpyaml | -hyaml | To generate a configuration file template for setting properties. |
 
-Additional resources are available at [Synopsys Software Integrity Community](https://community.synopsys.com).
+Additional resources are available at [Black Duck Community](https://community.blackduck.com/s/my-support-home).
```

Comment:
Is the additional 's/my-support-home' correct?

Reply:
Yep, that's the new URL they want for the support main entry page.